_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -9,84 +9,95 @@ nav_order: 50
 Introduced 2.9
 {: .label .label-purple }
 
-Create, manage, and take action on your alerts in a single, consolidated view and identify and resolve issues quickly. Use the **Dashboard** interface to:
+Create, manage, and take action on your alerts in a single, consolidated view and identify and resolve issues quickly through the **Dashboards** application in OpenSearch Dashboards. 
 
-- Set up, add, and adjust rules and conditions that trigger alerts and notifications.
-- Create graphs that show trends and patterns and build intuitive dashboards to stay informed of important metrics and data points in real time.
-- Monitor your alerts in one place with at-a-glance views.
-
-The following image gives you a snapshot of the Dashboard interface. 
-
-<img src="{{site.url}}{{site.baseurl}}/images/dashboards/alerting-dashboard.png" alt="Example alerting visualization" width="800" height="800">
+<img src="{{site.url}}{{site.baseurl}}//images/dashboards/dashboards-app.png" alt="Alerting interface in OpenSearch Dashboards" width="250"/>
```

Comment:
Note to self: Add an image showing an alerting visualization.

Reply:
SME: Do we have example alerting dashboards and visualizations? Can I access them through the playground, or do I need a test environment?

Reply:
Can we reuse the old image? This cannot be accessed through playground